packages/plugin-conversation/src/process-image.browser.js

```diff
@@ -94,7 +92,7 @@ export default function processImage({file, thumbnailMaxWidth, thumbnailMaxHeigh
         },
         file);
       const parts = canvas.toDataURL(`image/png`).split(`,`);
-      const byteString = base64.decode(parts[1]);
+      const byteString = atob(parts[1]);
```

Comment:
Add a comment about why we're not using base64. Otherwise, I imagine I'll see this in six months and try to "fix" it.